kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -424,8 +449,37 @@ public void onClick(View v) {
             }
         }
 
+        mainDivider = list.findViewById(R.id.idMainDividerLine);
         mainEditTextLinearLayout = (LinearLayout) list.findViewById(R.id.main_edit_text_linear_layout);
         individualMessageSendLayout = (LinearLayout) list.findViewById(R.id.individual_message_send_layout);
+
+        feedbackDisplayLayout = list.findViewById(R.id.idRelativeLayoutFeedbackDisplay);
+        textViewRestartConversation = feedbackDisplayLayout.findViewById(R.id.idFeedbackRestartConversation);
+        textViewfeedbackComment = feedbackDisplayLayout.findViewById(R.id.idFeedbackComment);
+        imageViewFeedbackRating = feedbackDisplayLayout.findViewById(R.id.idRatingImage);
+        constraintLayoutFeedbackTopLayout = feedbackDisplayLayout.findViewById(R.id.idFeedbackTopLayout);
+
+        Utils.printLog(getContext(), TAG, "onCreateView for "+TAG+" called.");
+
+        /**
+         * check if conversation is a resolved one, and display the respective feedback layouts
+         * also open the feedback input fragment if feedback isn't set
+         */
+        if(channel!=null && channel.getKmStatus() == Channel.CLOSED_CONVERSATIONS) {
```

Comment:
please review all code and do code formatting, not necessarily on just the line where the comment is.

Reply:
Use IDE code formatting option, IDE will format the whole file